lib/mayaUsd/ufe/StagesSubject.h

```diff
@@ -88,7 +88,7 @@ class MAYAUSD_CORE_PUBLIC StagesSubject : public TfWeakBase
     void onStageInvalidate(const MayaUsdProxyStageInvalidateNotice& notice);
 
     // Array of Notice::Key for registered listener
-#if UFE_PREVIEW_VERSION_NUM >= 2029
+#if UFE_PREVIEW_VERSION_NUM >= 2025
```

Comment:
@ppt-adsk  Based on our internal discussion,  lowering the version to cover existing shipped PR.